CodeMaid/Helpers/CodeItemTypeComparer.cs

```diff
@@ -139,6 +168,23 @@ private static int CalculateReadOnlyOffset(BaseCodeItem codeItem)
             return codeItemField.IsReadOnly ? 0 : 1;
         }
 
+        private static string NormalizeName(BaseCodeItem codeItem)
+        {
+            string name = codeItem.Name;
+            var interfaceItem = codeItem as IInterfaceItem;
+            if ((interfaceItem != null) && interfaceItem.IsExplicitInterfaceImplementation)
```

Comment:
Initialially I wrote it as `if (interfaceItem?.IsExplicitInterfaceImplementation == true)` but didn't like the lifting to nullable and also it's (for me!) weird to have if statements explicitly comparing for true, as it looks redundant.